regulations/generator/node_types.py

```diff
@@ -45,6 +49,22 @@ def label_to_text(label, include_section=True, include_marker=False):
             or _l2t_section(label, include_section, include_marker))
 
 
+_MARKERLESS_REGEX = re.compile(r'p\d+')
+
+
+def _paragraph_tail(label_parts, join_with, prefix='', suffix=''):
```

Comment:
This doesn't have any tests associated directly with it, which means it's hard to know what it's supposed to do.